eng/sdk/sync_sdk.py

```diff
@@ -57,15 +57,16 @@ def get_generated_folder_from_artifact(module_path: str, type: str, artifact: st
 
 def update_sdks():
     for tsp_location_file in glob.glob(os.path.join(sdk_root, 'sdk/*/*/tsp-location.yaml')):
-        tsp_location_path = os.path.dirname(tsp_location_file)
-        artifact = os.path.basename(tsp_location_path)
+        module_path = os.path.dirname(tsp_location_file)
+        artifact = os.path.basename(module_path)
 
         if artifact in skip_artifacts:
             continue
 
-        module_path = os.path.join(sdk_root, os.path.dirname(tsp_location_path))
```

Comment:
It was a bug. Above already took the `dirname` of the path. No need to do this a 2nd time.